addon/components/expandable-responsive-leaderboard.js

```diff
@@ -1,13 +1,24 @@
 import Component from '@ember/component';
 import { computed } from '@ember/object';
 import layout from '../templates/components/expandable-responsive-leaderboard';
+import config from 'ember-get-config';
+import DEFAULT_NETWORK_CODE from 'nypr-ads/defaults';
 
 export default Component.extend({
   layout,
 
-  network: '6483581',
+  network: DEFAULT_NETWORK_CODE,
   unitSuffix: 'leaderboard',
   slot: computed('network', 'site', function() {
-    return `/${this.get('network')}/leaderboard/${this.get('site')}_${this.get('unitSuffix')}`;
-  })
+    return `/${this.get('network')}${this.get('prefix')}/leaderboard/${this.get('site')}_${this.get('unitSuffix')}`;
+  }),
+  init() {
+    this._super(...arguments);
+    if (config.nyprAds) {
+      this.set('network', config.nyprAds.networkCode || DEFAULT_NETWORK_CODE);
+      this.set('prefix', config.nyprAds.prefix ? `/${config.nyprAds.prefix}` : null);
+    } else {
+      this.set('network', DEFAULT_NETWORK_CODE);
+    }
```

Comment:
these are all build-time values, so you don't need to set them in the init. they can be set directly as class variables with the same one-line booleans.

Reply:
The linter threw warnings about leaking state when i tried this. I know it should all resolve to strings but I couldn't make the linter see things my way.